src/common/src/Modal/Modal.tsx

```diff
@@ -20,7 +21,7 @@ export interface IModal extends React.FunctionComponent<Props> {
 	Actions: React.FunctionComponent<IModalActionsProps> | any;
 }
 
-const Modal: IModal = ({ id, children, classnames, onClose, size = 'md', ...rest }: Props) => {
+const Modal: IModal = ({ id, children, classnames, onClose, size = 'md', disablePortal, ...rest }: Props) => {
```

Comment:
Why destruct it instead of letting it be part of ...rest?

Reply:
It needs to be added as an overload in the return `disablePortal={disablePortal}` on line 59, or else it doesn't seem to get passed through.